curry_bot.py

```diff
@@ -82,16 +92,57 @@ async def bingo(ctx):
         await ctx.send(curry_message("...done. Room created at URL: {} with password: {}\nGo do your best! Don't stumble!".format(room_url, italics(password))))
 
 
-@client.command(description="Type '!rando' to fetch the current seed link runners are playing on. Type '!rando <link>' to overwrite current seed with a new one of your choice.", brief="Get rando seed in progress")
+@client.command(description="Type '!rando' to fetch the current seed link runners are playing on.\nType '!rando <link>' to overwrite current seed with a new one of your choice.\nType '!rando presets' for a list of presets.\nType '!rando <preset>' to generate {} seeds of the given preset.".format(SEEDS_TO_GENERATE), brief="Make or get current rando seeds")
 async def rando(ctx, *args):
     global RANDO_LINKS
+    print_links = True
     if len(args) >= 1:
-        RANDO_LINKS = args
-        await ctx.send(curry_message("Rando seed links updated"))
-    await ctx.send(curry_message("Current rando seed links:"))
-    for i, link in enumerate(RANDO_LINKS):
-        await ctx.send(curry_message("Seed {}: {}".format(i+1, link)))
-
+        # Match on the start to make this command respond to 'presets' as well
+        if args[0].startswith('preset'):
+            print_links = False
+            for preset, description in RANDO_PRESETS.items():
+                await ctx.send(curry_message("Preset: {}, Description: {}".format(preset, description)))
```

Comment:
Can bulk together in one line-separated message, to reduce number of sends

Reply:
Fixed in 1ef78ba.